entities/src/referee.rs

```diff
@@ -0,0 +1,50 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.10.6
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "referee")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub used_referral_code: String,
```

Comment:
should this be a foreign key to the referrer table? then we don't need the user id on this table (though it does make the queries simpler)